.wiki/_DV/Laws/StandardOperatingProcedure.txt

```diff
@@ -420,3 +420,30 @@ In order for a medical evaluation to be considered lawfully applicable, the CMO
 
 ==Quarantine==
 Medical staff are afforded the right to quarantine individuals that currently suffer from infectious or contagious diseases, and may confine them within pre-designated quarantine areas in the course of treatment, until the point that they are no longer infectious or harmful to the rest of the crew.
+=Engineering Regulations=
+===Wiring===
+All wiring must be concealed from public for their safety.
+In non public areas, catwalk is recommended to be placed for safety and mobility of workers.
+
+Wiring is to be held straight and tidy, without unnecessary looping, to avoid clumping, short circuiting and trpping hazard for the crew.
```

Comment:
tripping typo

unnecessary looping is **good** as it provides redundancy
clarify short circuiting to be with SMESes as that is the only thing you can short circuit